src/manager/manager.c

```diff
@@ -914,6 +914,7 @@ static int manager_name_owner_changed(sd_bus_message *m, void *userdata, UNUSED
 }
 
 bool manager_start(Manager *manager) {
+        hirte_log_infof("Starting hirte manager %s", CONFIG_H_HIRTE_VERSION);
```

Comment:
I'd prefer to use just `hirte` instead of  `hirte manager`, `hirte` is the official service name and also in the past we tried to remote the word *manager* from any hirte description and documents